README.md

```diff
@@ -2,6 +2,14 @@
 
 *for plotting data with units seamlessly in Julia*
 
+## Deprecation
+
+As of Plots.jl `v1.34.1`, UnitfulRecipes is included in Plots, and is no longer
```

Comment:
```suggestion
As of Plots.jl `v1.34.2`, UnitfulRecipes is included in Plots, and is no longer
```

Actually it's `1.34.2` https://github.com/JuliaPlots/Plots.jl/commit/e0c58fbe805666e65536d926caa8a459f9376028.